gnucash/gnome/window-reconcile.c

```diff
@@ -2288,10 +2301,10 @@ recnFinishCB (GtkAction *action, RecnWindow *recnData)
     recnData->delete_refresh = TRUE;
     account = recn_get_account (recnData);
 
-    xaccAccountBeginEdit (account);
+    acct_traverse_descendants (account, xaccAccountBeginEdit);
```

Comment:
It's indeed too smart for me. BeginEdit and CommitEdit are common database terminology. At least I would miss them.

Reply:
There's no plan to remove BeginEdit and CommitEdit.